File: Problems/0771. Jewels and Stones/numJewelsInStones.java
class Solution {
    public int numJewelsInStones(String jewels, String stones) {
        int jlen = jewels.length();
        int slen = stones.length();
        int count = 0;
        for(int i = 0; i < jlen; i++) {
            char jewel = jewels.charAt(i);
            for(int j = 0; j < slen; j++) {
                if(jewel == stones.charAt(j)) {
                    count++;
                }
            }
        }
        return count;
    }
}

File: Problems/0771. Jewels and Stones/numJewelsInStones.js
/**
 * @param {string} jewels
 * @param {string} stones
 * @return {number}
 */
var numJewelsInStones = function(jewels, stones) {
    const jewelArray = jewels.split("");
    const stoneArray = stones.split("");
    let jewelCount = 0;
    stoneArray.forEach(stone => {
        jewelArray.includes(stone) && jewelCount++;
    })
    return jewelCount;
};